src/index.js

```diff
@@ -79,4 +79,4 @@ tasks.forEach((task) => {
 });
 saveData();
 document.querySelector('.add-btn').addEventListener('click', addToList);
-document.querySelector('.removeButton').addEventListener('click', removeList);
\ No newline at end of file
+document.querySelector('.removeButton').addEventListener('click', removeList);
```

Comment:
- [ ] All hardcoded items from the tasks array need to be  removed.
![Uploading image.png…]()